radialIndicator.js

```diff
@@ -281,29 +286,41 @@
             //limit the val in range of minumum and maximum value
             val = val < minVal ? minVal : val > maxVal ? maxVal : val;
 
-            var precision = indOption.precision != null ? indOption.precision : 0,
+            var frameNum = indOption.frameNum || (indOption.percentage ? 100 : 500),
+                legacyDecimals = (maxVal - minVal / frameNum).countDecimals(), // precision default legacy support
```

Comment:
It will always do maxVal - (minVal / frameNum) as division has higher precedence. I don't think we want that. I had the same issue on the last logic. 
```suggestion
                legacyDecimals = ((maxVal - minVal) / frameNum).countDecimals(), // precision default legacy support
```

Reply:
Also this might break due to floating point precision in JS.

`(1 - .3) / 10` will give `0.006999999999999999` Instead of `0.007`